java/tests/src/test/scala/ironrust/FullIntegrationTest.scala

```diff
@@ -59,7 +59,13 @@ class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
       // any class that we can't implement equals and hashCode for must be in this list
       val iclExclude = List("AssociationType", "IronOxide", "BlindIndexSearch")
       val currentPath = java.nio.file.Paths.get("").toAbsolutePath.getParent.toString
-      val javaFiles = new java.io.File(s"$currentPath/java/com/ironcorelabs/sdk").listFiles
+      val fileFilter = new java.io.FileFilter {
+        override def accept(pathname: java.io.File): Boolean =
+          pathname.getName.startsWith("ironoxide-java") && pathname.listFiles.map(file => file.getName).contains("out")
+      }
+      val ironoxideJavaFolder = new java.io.File(s"$currentPath/../target/debug/build/").listFiles(fileFilter)
+      ironoxideJavaFolder.length shouldBe 1
```

Comment:
This error message is going to be really bad for the person who runs into it.

I suggest giving a custom error message about too many java files.

Reply:
Ideally the `build.sbt` error handling would stop us from even getting here, but I added custom error messages for the two failure cases

Reply:
Thanks for that. The build.sbt error would only appear when you reload SBT. If you did cargo clean in between you wouldn't get that error.